src/utils/helpers/GraphHelpers.ts

```diff
@@ -86,36 +71,149 @@ export function selectHighLightedNodes() {
 }
 
 export function selectAllNodes() {
-    return select('#labels').selectAll<SVGGElement, DependencyNode>('g');
+    return select(Selectors.LABELS).selectAll<SVGGElement, DependencyNode>('g');
+}
+
+export function selectHighlightBackground() {
+    return select(Selectors.HIGHLIGHT_BACKGROUND);
+}
+
+function selectDetailsButtonWrapper() {
+    return select(Selectors.DETAILS_BUTTON);
```

Comment:
Would it be possible to store a reference to those nodes instead querying for them?

Reply:
It's would be tricky to select those nodes once and then only keep references in consts - I would have to make sure that these nodes get created before graphHelpers module gets loaded somewhere in the app - which is at least very hard to achieve :)